app/models/amt/AMTConditionTable.scala

```diff
@@ -28,6 +33,29 @@ class AMTConditionTable(tag: Tag) extends Table[AMTCondition](tag, Some("sidewal
 object AMTConditionTable {
   val db = play.api.db.slick.DB
   val amtConditions = TableQuery[AMTConditionTable]
+  val amtAssignments = TableQuery[AMTAssignmentTable]
+  val max_num_condition_assignments = 5
+
+  def getVolunteerIdByConditionId(amtConditionId: Int): String = db.withTransaction { implicit session =>
+    val vId = amtConditions.filter(_.amtConditionId === amtConditionId).list.headOption.map(_.volunteerId)
```

Comment:
Can do "select volunteer_id" instead of "select *"...
amtConditions.filter(\_.amtConditionId === amtConditionId).map(\_.volunteerId).list.headOption